membership-common/src/test/scala/com/gu/memsub/subsv2/reads/ChargeListReadsTest.scala

```diff
@@ -135,17 +135,6 @@ class ChargeListReadsTest extends AnyFlatSpec {
 
   }
 
-  "product reads" should "not read any supporter as a Free member" in {
```

Comment:
no such thing as a free member now.  We have tests for other kinds of incorrect charge lists below so I deleted this one.